front/components/vaults/ContentActions.tsx

```diff
@@ -120,78 +120,87 @@ ContentActions.displayName = "ContentActions";
 type ContentActionsMenu = ComponentProps<typeof DataTable.Row>["moreMenuItems"];
 
 export const getMenuItems = (
+  canReadInVault: boolean,
+  canWriteInVault: boolean,
   dataSourceView: DataSourceViewType,
   contentNode: LightContentNode,
   contentActionsRef: RefObject<ContentActionsRef>
 ): ContentActionsMenu => {
-  if (isFolder(dataSourceView.dataSource)) {
-    return [
-      {
-        label: "Edit",
-        icon: PencilSquareIcon,
-        onClick: () => {
-          contentActionsRef.current &&
-            contentActionsRef.current?.callAction(
-              contentNode.type === "database"
-                ? "TableUploadOrEdit"
-                : "DocumentUploadOrEdit",
-              contentNode
-            );
-        },
+  const actions: ContentActionsMenu = [];
+
+  // Edit & Delete
+  if (canWriteInVault && isFolder(dataSourceView.dataSource)) {
```

Comment:
We should probably sort those menu items, right?

IMHO:
- View in
- View raw content
- Edit
- Delete